source/Calamari/Kubernetes/Helm/GitRepositoryValuesFileWriter.cs

```diff
@@ -15,12 +16,12 @@ public static IEnumerable<string> FindChartValuesFiles(RunningDeployment deploym
                                                                ILog log,
                                                                string valuesFilePaths,
                                                                bool logIncludedFiles = true)
-            => FindGitDependencyValuesFiles(deployment,
```

Comment:
I'm confused on the naming here now. The HelmTemplateValueSourcesParser is checking if the script source is a GitRepository and if it is calling GitRepositoryValuesFileWriter.FindChartValuesFiles. Am I right in understanding this is a normal project with a git sourced helm chart and the values file is brought in as a part of the git repository? My understanding might be way off here, really all I'm trying to check is have we checked all combinations of CAC, git sourced, and normal projects with this change?

Reply:
I haven't changed to flow here, just removed the check for the git dependency name for the "value from chart" path

The specific scenario is a CAC project where the helm chart comes from the Project git repo and the values file is source from the chart.

I have also documented a workaround (confirmed as working by the customer) which may negate the need to backport.

Reply:
The naming was confusing before anyway